examples/motoko_examples/factorial/canisters/motoko/main.mo

```diff
@@ -0,0 +1,19 @@
+actor Factorial {
+
+  // Calculate the product of all positive integers less than or equal to `n`.
+  public query func fac(n : Nat) : async Nat {
+
+    // We implement the recustion in a helper function that does not return
```

Comment:
`recustion` should be spelled `recursion`

Reply:
That was coming from the dfinity/examples codebase so I didn't even realize. I've fixed it though.